CMake/External_OpenCV.cmake

```diff
@@ -21,6 +24,23 @@ else()
   set(fletch_ENABLE_OpenCV_FFmpeg CACHE INTERNAL FALSE)
 endif()
 
+# Allow OpenCV's VTK option to be explicitly turned off
+if(fletch_ENABLE_VTK)
+  # By default turn opencv-VTK support off, even when building with VTK
+  option(fletch_ENABLE_OpenCV_VTK "Build OpenCV with VTK support" FALSE )
+  mark_as_advanced(fletch_ENABLE_OpenCV_VTK)
+else()
+  set(fletch_ENABLE_OpenCV_VTK CACHE INTERNAL FALSE)
```

Comment:
I think if we user CACHE INTERNAL FALSE here, we lose the ability to reenable it with the same cache. I believe we had the same issue with Darknet. I believe @aaron-bray came up with a decent working paradigm for this type of action. Hopefully he will chime in.